src/Pages/Explanation.tsx

```diff
@@ -19,6 +19,23 @@ class Explanation extends React.Component {
                 <div className="content">
                     <Header home={false} />
 
+                    <Row>
+                        <Col span={1}></Col>
+                        <Col span={22}>
+                            <Card title="Beta Studiengang" bordered={false} style={{ maxWidth: 1000 }}>
+                                <p>
+                                    Sobald wir einen neuen Studiengang hinzugefügt haben versuchen wir in Zusammenarbeit mit den entsprechenden Ansprechpartnern des
```

Comment:
Fehlt hier ein Komma?

"Sobald wir einen neuen Studiengang hinzugefügt haben, versuchen wir in Zusammenarbeit ..."

Reply:
true :D